### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by LabudSoft** [MilanLabuss/LearningGithub#1]
Please add a feature where I can upload and use my own image assets

**Comment by MilanLabuss** [comment]
Good Idea. Fork the repository and create it yourself and submit a pull request
